stable/restore/README.md

```diff
@@ -152,9 +152,12 @@ The following tables list the configurable parameters of the restore chart and t
 | ----- | ----------- | ------ |
 | `restore.type` | What type of restore to perform: [ "database" | "archive" ]. A "database" restore restarts the entire database at a previous state. An "archive" restore restores/repairs a SINGLE archive in a RUNNING database. | `"database"` |
 | `restore.target` | Where to restore `TO` | `{{ .Values.database.name }}` |
-| `restore.source` | Where to restore `FROM` [ backupset | url | `:latest` ] | `:latest` |
+| `restore.source` | Where to restore `FROM` [ backupset | url | `:latest` | `:group-latest` ] | `:latest` |
 | `restore.credentials` | Credentials to use for a URL source (user:password) | `""` |
 | `restore.autoRestart` | Whether to automatically restart the database and trigger the restore (true/false). Only valid for a "database" restore | `true` |
+| `restore.labels` | Process labels used to filter the complete set of archiveIds that should be restored which then defines the new state of the database upon restore. The setting works with NuoDB 4.2+ | `backup {{ .Values.cloud.cluster.name }}` |
```

Comment:
I would not use the helm template here. I would use the default rendered value.

Reply:
I will change it.